Makefile

```diff
@@ -58,6 +58,8 @@ install-full-windows: install install-mitie
 	poetry install -E spacy -E transformers -E jieba
 
 install-docs:
+	# FIXME: poetry doesn't support installing from a git subdirectory
+	poetry run pip install git+https://github.com/RasaHQ/pydoc-markdown.git@23d825f#subdirectory=pydoc-markdown
```

Comment:
really not idea but not sure there is a workaround

Reply:
can we just publish and install from our pypi.rasa.com?

Reply:
happy to do it; can you share credentials in 1password?